components/movementsensor/client_test.go

```diff
@@ -177,7 +182,7 @@ func TestClient(t *testing.T) {
 		test.That(t, props1.LinearVelocitySupported, test.ShouldResemble, props.LinearVelocitySupported)
 		test.That(t, injectMovementSensor.PropertiesFuncExtraCap, test.ShouldResemble, map[string]interface{}{"foo": "bar"})
 
-		acc1, err := gps1Client.Accuracy(context.Background(), map[string]interface{}{"foo": "bar"})
+		acc1, _, _, _, _, err := gps1Client.Accuracy(context.Background(), map[string]interface{}{"foo": "bar"})
```

Comment:
if these fields are optional, can go not return them like in another language e such as python? Let's ask someone with more in depth go knowledge.